src/control-center/src/main.cpp

```diff
@@ -1,32 +1,55 @@
 #include <iostream>
 #include <ControlCenter.h>
 #include <Serializer.h>
+#include <Logger.h>
+#include <ConfigReader.h>
 #include <csignal>
 #include <chrono>
 #include <thread>
 
 using namespace std;
 
-Serializer s;
-ControlCenter cc(s);
-bool quit = false;
+namespace {
+    static const int8_t BROADCAST_SENSORS_INTERVAL = 3;
+    ControlCenter *g_cc;
```

Comment:
To powinno być `volatile` (w sensie wskaźnik, nie klasa: `ControlCenter * volatile g_cc`).